caas/docker/versions.tf

```
@@ -2,6 +2,7 @@ terraform {
   required_providers {
     datadog = {
       source = "terraform-providers/datadog"
```

Source for datadog provider has changed:
```suggestion
      source = "DataDog/datadog"
```
See https://registry.terraform.io/providers/DataDog/datadog/latest/docs